vllm/config.py

```diff
@@ -368,9 +368,35 @@ def get_num_attention_heads(self,
         return num_heads // parallel_config.tensor_parallel_size
 
     def get_num_layers(self, parallel_config: "ParallelConfig") -> int:
-        total_num_hidden_layers = self.hf_text_config.num_hidden_layers
+        total_num_hidden_layers = getattr(self.hf_text_config,
+                                          "num_hidden_layers", 0)
         return total_num_hidden_layers // parallel_config.pipeline_parallel_size
 
+    def contains_seqlen_agnostic_layers(
```

Comment:
nit: do we need two methods for this op? if it's just for code cleanliness let's make the other one private (`_get_num_seqlen_agnostic_layers`)

Reply:
nit: suggest small docstring explaining an example when this would be true (e.g. Mamba/SSM models)

Reply:
Yeah, changed `_get_num_seqlen_agnostic_layers` to private and added a small docstring